src/common/components/hosts/DiskRole.tsx

```diff
@@ -88,7 +92,7 @@ const DiskRoleDropdown: React.FC<DiskRoleDropdownProps> = ({
     [setOpen, setDisabled, onDiskRole, host.id, disk.id],
   );
 
-  const currentRoleLabel = getCurrentDiskRoleLabel(disk, installationDiskId);
+  const currentRoleLabel = getCurrentDiskRoleLabel(disk, installationDiskId, t);
```

Comment:
There is a second usage of this function which needs to be updated too. It fails when clicking on a host in the table.

I suggest an alternative change which avoids having to call the function twice and passing the extra parameter. Pass `currentRoleLabel` to `DiskRoleDropdown` instead (more changes are needed, please check)

```
const DiskRole = ({ host, disk, installationDiskId, isEditable, onDiskRole }: DiskRoleProps) => {
  const { t } = useTranslation();
  const currentRoleLabel = getCurrentDiskRoleLabel(disk, installationDiskId, t);

  if (isEditable && disk.id !== installationDiskId && onDiskRole) {
    return (
      <DiskRoleDropdown
        host={host}
        disk={disk}
        currentRoleLabel={currentRoleLabel}
        onDiskRole={onDiskRole}
      />
    );
  }
  return <>{currentRoleLabel}</>;
};
```

Reply:
I've changed the DiskRole constant as you suggested